data/moves.js

```diff
@@ -1294,19 +1295,11 @@ let BattleMovedex = {
 						this.add('-miss', pokemon, target);
 						return false;
 					}
-					let moveData = {
-						id: 'bide',
-						name: "Bide",
-						accuracy: true,
+					Object.assign(move, {
```

Comment:
I'm pretty sure overwriting the `move` object here is an actual bug. Leave it as a copy.